classes/amity.py

```diff
@@ -1,45 +1,515 @@
+    changes = False
+    loaded_database = ""
     offices = []
+    livingspaces = []
+    fellows = []
+    staffs = []
+    waiting_list = {
+        "office": [],
+        "livingspace": []
+    }
 
     def create_room(self, room_type, room_names):
         """A method that is used to create a room"""
-        pass
+        room_type = room_type.lower()
+        message = ""
+        for room_name in room_names:
+            room_name = room_name.lower()
+            if room_name not in [room.room_name
+               for room in (self.offices + self.livingspaces)]:
+                if room_type == "office":
+                    new_office = Office(room_name)
+                    self.offices.append(new_office)
+                    self.changes = True
+                    message += "{} added successfully!\n".format(
+                     room_name.title())
+                elif room_type in ["living_space", "livingspace"]:
+                    new_living_space = LivingSpace(room_name)
+                    self.livingspaces.append(new_living_space)
+                    self.changes = True
+                    message += "{} added successfully!\n".format(
+                     room_name.title())
+                else:
+                    message += "Invalid room type. " + \
+                      "A room can either be of type office or living_space!\n"
+            else:
+                message += "Room {} already exists!\n".format(
+                 room_name.title())
+        return message
 
-    def add_person(self, person_name, person_type, accommodate):
+    def add_person(self, person_name, person_type, wants_accommodation):
         """A method that is used to add a person into the system"""
-        # reject accommodate staff y/n for staff members
-        pass
+        is_digit = any(char.isdigit() for char in person_name)
+        if is_digit:
+            return "Invalid name. Use letters only"
+        else:
+            person_name = person_name.title()
+            person_type = person_type.lower()
+            if person_type not in ["fellow", "f", "staff", "s"]:
+                return "Invalid role. You can either be a fellow/staff."
+            else:
+                wants_accommodation = (wants_accommodation.lower()
+                                       if wants_accommodation is not None
+                                       else "no")
+                if wants_accommodation not in ["yes", "y", "n", "no"]:
+                    return "Invalid accomodate option. " + \
+                     "It can either be yes or no."
+                else:
+                    return self.save_person(person_name, person_type,
+                                            wants_accommodation)
+
+    def save_person(self, person_name, person_type, wants_accommodation):
+        person_name = person_name.title()
+        if person_type in ["staff", "s"]:
+            if wants_accommodation in ["yes", "y"]:
```

Comment:
Allow staff to be added but don't give them accommodation and give the warning message to the user.